src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  /* padding: 0; */
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
You need to use margin-right for all items besides last one. Use `pseudo-element`

Reply:
if i do it - file will not pass a tests